### PR TITLE
Remove a katib-webhook-cert Secret from components

### PR DESCRIPTION
**Pull request by tenzen-y**
**What this PR does / why we need it**:
I removed a katib-webhook-cert Secret from the components directory to avoid startup issues in the cert-manager installation.

### Need katib-webhook-cert

- [x] katib-external-db
- [x] katib-leader-election
- [x] katib-standalone
- [x] katib-standalone-postgres

### **_NOT_** to need katib-webhook-cert

- [x] katib-cert-manager
- [x] katib-openshift
- [x] katib-with-kubeflow

Also, I confirmed built manifests like this:

```shell
$ kustomize build manifests/v1beta1/installs/katib-cert-manager/ | grep -C2 "name: katib-webhook-cert"
kind: Certificate
metadata:
  name: katib-webhook-cert
  namespace: kubeflow
spec:
$ kustomize build manifests/v1beta1/installs/katib-external-db/ | grep -C2 "name: katib-webhook-cert"
kind: Secret
metadata:
  name: katib-webhook-cert
  namespace: kubeflow
---
$ kustomize build manifests/v1beta1/installs/katib-leader-election/ | grep -C2 "name: katib-webhook-cert"
kind: Secret
metadata:
  name: katib-webhook-cert
  namespace: kubeflow
---
$ kustomize build manifests/v1beta1/installs/katib-openshift/ | grep -C2 "name: katib-webhook-cert"
    prometheus.io/scheme: http
    prometheus.io/scrape: "true"
    service.beta.openshift.io/serving-cert-secret-name: katib-webhook-cert
  labels:
    katib.kubeflow.org/component: controller
$ kustomize build manifests/v1beta1/installs/katib-standalone/ | grep -C2 "name: katib-webhook-cert"
kind: Secret
metadata:
  name: katib-webhook-cert
  namespace: kubeflow
---
$ kustomize build manifests/v1beta1/installs/katib-standalone-postgres/ | grep -C2 "name: katib-webhook-cert"
kind: Secret
metadata:
  name: katib-webhook-cert
  namespace: kubeflow
---
$ kustomize build manifests/v1beta1/installs/katib-with-kubeflow/ | grep -C2 "name: katib-webhook-cert"
kind: Certificate
metadata:
  name: katib-webhook-cert
  namespace: kubeflow
spec:
```

**Which issue(s) this PR fixes** _(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)_:
Fixes #

**Checklist:**

- [ ] [Docs](https://www.kubeflow.org/docs/components/katib/) included if any changes are user facing

### CODE REVIEW

File: manifests/v1beta1/installs/katib-external-db/kustomization.yaml [hunk 1]
```diff
@@ -15,6 +15,8 @@ resources:
   - ../../components/ui/
   # Katib webhooks.
   - ../../components/webhook/
+  # Secret for webhooks certs.
+  - ./katib-webhook-cert.yaml
```
Comment (andreyvelich) [inline manifests/v1beta1/installs/katib-external-db/kustomization.yaml:19]: I think, we can simplify this as follows in the Kustomization file:
```
secretGenerator:
  - name: katib-webhook-cert
    options:
      disableNameSuffixHash: true
```

WDYT @tenzen-y ?

Comment (tenzen-y) [reply manifests/v1beta1/installs/katib-external-db/kustomization.yaml:19]: It's a good point! Thanks!

Comment (tenzen-y) [reply manifests/v1beta1/installs/katib-external-db/kustomization.yaml]: Done.

**Pull request merged**